policy/modules/system/selinuxutil.te

```diff
@@ -249,6 +249,7 @@ read_lnk_files_pattern(newrole_t, default_context_t, default_context_t)
 
 kernel_read_system_state(newrole_t)
 kernel_read_kernel_sysctls(newrole_t)
+kernel_getattr_proc(newrole_t)
```

Comment:
or should this be dontaudit'ed?

Reply:
I go back and forth on this. Strictly speaking, dontaudit it, if it isn't required.